src/adapters/nfts/index.ts

```diff
@@ -68,10 +69,10 @@ export function fromDBNFTToNFT(dbNFT: DBNFT): NFT {
     category: dbNFT.category,
     chainId: getNetworkChainId(dbNFT.network),
     contractAddress: dbNFT.contract_address,
-    createdAt: dbNFT.created_at,
+    createdAt: fromSecondsToMilliseconds(dbNFT.created_at),
```

Comment:
should it be parsed to number first?
```suggestion
    createdAt: fromSecondsToMilliseconds(Number(dbNFT.created_at)),
```